documentation/books/evaluating/quickstart.html

```diff
@@ -0,0 +1,962 @@
+<!DOCTYPE html>
```

Comment:
I think we should not commit the generated HTML. We should generate it in the CI and push it to the website. We can do that in followup PR. But we should not commit this here I think.